pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
```

Comment:
Did you wanna add any keywords to make it easier to lookup?

Reply:
Added "tqdm" like from `setup.py`. Any suggestions?

Reply:
pubsub - python progress bar - "publish-subscribe pattern" (since that's how the wikipedia article describes the messaging system)

Any others that uniquely qualify relation to parent package and expand differences